velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -175,5 +175,38 @@ TEST_F(DateTimeFunctionsTest, makeDate) {
   EXPECT_EQ(makeDate(2023, 3, 29), expectedDate);
 }
 
+TEST_F(DateTimeFunctionsTest, lastDay) {
+  const auto lastDayFunc = [&](const std::optional<Date> date) {
+    return evaluateOnce<Date>("last_day(c0)", date);
+  };
+
+  const auto lastDay = [&](const std::string& dateStr) {
+    Date d0;
+    parseTo(dateStr, d0);
+    return lastDayFunc(d0);
+  };
+
+  const auto parseDateStr = [&](const std::string& dateStr) {
+    Date d0;
+    parseTo(dateStr, d0);
```

Comment:
Thank. I wonder why parseTo is not just returning Date. Seems quite inconvenient to use. CC: @aditi-pandit

Reply:
It would be nice to change parseTo to return Date: `Date d = parseTo("2022-10-13")`. Would appreciate a PR.